com.unity.render-pipelines.universal/Runtime/NativeRenderPass.cs

```diff
@@ -368,12 +381,15 @@ internal void ExecuteNativeRenderPass(ScriptableRenderContext context, Scriptabl
                 {
                     for (int i = 0; i < attachmentIndicesCount; ++i)
                     {
-                        attachmentIndices[i] = renderPass.m_InputAttachmentIndices[i];
+                        attachmentIndices[i] = renderPass.m_ColorAttachmentIndices[i];
                     }
                 }
 
                 if (validPassCount == 1 || currentMergeablePasses[0] == currentPassIndex) // Check if it's the first pass
                 {
+                    if (PassHasInputAttachments(renderPass)) //First ScriptableRenderPass cannot have input attachments - something is wrong
+                        return;
```

Comment:
Should we have an assert instead of failing silently?

Reply:
Yeah, at least a warning should be thrown. Will fix tomorrow